common/changes/@itwin/appui-react/Linas-updating-rimraf-arguments-and-deps_2025-02-14-09-12.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/appui-react",
+      "comment": "Upgrade to Node version 20",
```

```suggestion
      "comment": "",
```

Keep these comments empty, as this change has no impact to package consumers (leaving a single comment, but there are multiple change files).

---

Removed comment in this and other change files